portal/sign/assets/signer_modal.tpl.php

```diff
@@ -8,37 +8,59 @@
  * @copyright Copyright (c) 2016-2019 Jerry Padgett <sjpadgett@gmail.com>
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
-require_once dirname(__FILE__) . "/../../../vendor/autoload.php";
-require_once(dirname(__FILE__) . '/../../../library/sql.inc');
 
-$isAdmin = !empty($_GET['isAdmin']) ? 'checked' : '';
+require_once(dirname(__FILE__) . "/../../../src/Common/Session/SessionUtil.php");
+OpenEMR\Common\Session\SessionUtil::portalSessionStart();
+
+$is_portal = isset($_SESSION['portal_init']) ? true : $_GET['isPortal'];
+if (!$is_portal) {
+    session_destroy();
+    require_once(dirname(__FILE__) . '/../../../interface/globals.php');
+} else {
+    require_once dirname(__FILE__) . "/../../verify_session.php";
+}
+
+$cuser = js_escape(isset($_SESSION['authUser']) ? $_SESSION['authUser'] : "-patient-");
+$cpid = js_escape(isset($_SESSION['pid']) ? $_SESSION['pid'] : "0");
+$api_id = isset($_SESSION['api_csrf_token']) ? $_SESSION['api_csrf_token'] : ''; // portal doesn't do remote
 
 $msg1 = xlt('Show Current Signature On File');
 $msg2 = xlt('As appears on documents');
 $msg3 = xlt('Is Authorizing Signature');
 $msg4 = xlt('Sign Above');
 $msg5 = xlt('Clear');
 $msg6 = xlt('Acknowledge Electronic Signature');
-
-$sign_type = 'patient-signature';
-if ($isAdmin) {
-    $sign_type = 'admin-signature';
-}
-// short & sweet
+$msg7 = xlt('Signatory');
+$msg8 = xlt('Signatory did not sign or Signer was closed.');
+$msg9 = xlt('Waiting for Signature on Signer Pad');
+$msg10 = xlt('Please provide a signature first');
+$msg11 = xlt('Signer Pad Checked In and Available');
+$msg12 = xlt('Transaction Failed');
+// why tempt fate 13 a no no....
+$msg14 = xlt("A Remote Signing Device is not answering.") . "<br><h4>" . xlt("Using this device until remote becomes available.") . "</h4>";
+$msg15 = xlt("Remote is Currently Busy");
+// module translations
+$vars = "<script>const msgSignator='" . $msg7 . "';const msgNoSign='" . $msg8 . "';const msgWaiting='" . $msg9 . "';const msgBusy='" . $msg15 . "';</script>\n";
+$vars .= "<script>const msgNeedSign='" . $msg10 . "';const msgCheckIn='" . $msg11 . "';const msgFail='" . $msg12 . "';const msgAnswering='" . $msg14 . "';</script>\n";
+$vars .= "<script>const apiToken='" . js_escape($api_id) . "';</script>\n";
```

Comment:
In line above the, the single quotes should be removed around the js_escape($api_id) call, since js_escape will add them.